### PR TITLE
Bug/fix pushing custom code art 2504

### PR DESCRIPTION
**Pull request by archan937**
This pull request contains the following:

* Correct all-users.js example
* Add support to customize webpack.config.js
* Restore pushing custom code by using Webhead
* Implement 2FA

### CODE REVIEW

File: assets/functions/packer/webpack.config.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 module.exports = {
+  // WARNING: Do not change this section
   entry: './src/index.js',
```
Comment (stephankaag) [inline assets/functions/packer/webpack.config.js:3]: @archan937 Maybe add `target: node` to the default webpack.config.js?

https://webpack.js.org/configuration/target/

Comment (archan937) [reply assets/functions/packer/webpack.config.js:3]: Done, thank.

File: src/bb-functions-publish.ts [hunk 2]
```diff
@@ -205,10 +204,9 @@ const publishFunctions = async (
     {},
   );
 
-  let promises: Promise<void | string | object | null>[] = Object.keys(
-    metaData,
-  ).map(
-    (name: string): Promise<string | object | null> => {
+  await Object.keys(metaData).reduce(
```
Comment (dehlak) [inline src/bb-functions-publish.ts:207]: I would use something like this:

`Promise.all(Object.keys(metaData).map(m => new Promise(.....))`

Comment (archan937) [reply src/bb-functions-publish.ts]: Thanks for the tip. I have changed it accordingly.

**Review by dehlak — APPROVED** [review]

**Pull request merged**